dashboarddoc.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: dashboard
+permalink: /dashboarddoc/
+---
+
+User Dashboard Interaction
+==========================
+{: .anchor}
+
+This document illustrates how a Solar Forecast Arbiter user may use the
+Dashboard to complete the following actions:
+
+-   [Creating a new Site](#creating-a-site).
+
+-   [Creating a new Observation or Forecast](#creating-an-observation-or-forecast).
+
+-   [Uploading Data to an Observation or Forecast](#uploading-data).
+
+When a user is filling out a form, a tooltip describing the information
```

Comment:
this seems out of place. I suggest putting it at the end of "Create a new Site". I know it's not specific that section, but I don't think the reader is ready for the information at this point.